### PR TITLE
Add checklist to PRs

### PR DESCRIPTION
**Pull request by Eddasol**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Ready for review checklist:
+- [ ] A self-review has been performed
+- [ ] All commits run individually
+- [ ] All TODO, console.log or other temporary changes have been removed
```
Comment (Christdej) [inline .github/PULL_REQUEST_TEMPLATE.md:4]:
```suggestion
- [ ] Temporary changes have been removed, like console.log, etc.
- [ ] Unnecessary comments removed, like TODO, commented code, etc.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+- [ ] The PR has been tested locally
+- [ ] A test have been written
+  - [ ] This change doesn't need a new test
+- [ ] Relevant issues are linked
+- [ ] Remaining work is documented in issues
+  - [ ] There is no remaining work from this PR that require new issues
```
Comment (Christdej) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: Should we add
`- [ ] All pipelines run successfully`
Or perhaps we cannot check this off when creating PR? (depends on if this list is used on PR create or PR ready for review)

**Review by Christdej — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (anetteu) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: I suggest to add something similar to 

```
- [  ] The changes does not introduce dead code as unused imports, functions etc. 

```

Comment (Eddasol) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: No we need this when we also have the pipelines further down that gives the same check. Should be included in private repos though

Comment (Eddasol) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: image
The checklist will appear in the marked area and cannot be checked until after the pr is created

Comment (Christdej) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: Then I agreed we dont add this =)

**Review by Christdej — APPROVED** [review]

**Pull request merged**